packages/progress-bar/src/ProgressBar/ProgressBar.styles.ts

```diff
@@ -246,45 +255,38 @@ export const getBarFillStyles = ({
 
   let addOnStyles;
 
-  const determinate = getDeterminateBarFillStyles({
-    theme,
-    color,
-    disabled,
-    width: getPercentage(value, maxValue),
-  });
+  const determinate =
+    isDefined(value) &&
+    getDeterminateFillStyles({
+      theme,
+      variant,
+      disabled,
+      width: getPercentage(value, maxValue),
+    });
 
-  const determinateAnimated = getDeterminateAnimatedBarFillStyles({
+  const determinateAnimated = getDeterminateAnimatedFillStyles({
     theme,
-    color,
+    variant,
     disabled,
   });
 
-  const indeterminate = getIndeterminateBarFillStyles({ theme, color });
+  const indeterminate = getIndeterminateFillStyles({ theme, variant });
 
   switch (animationMode) {
     case AnimationMode.Transition:
-      addOnStyles = cx(indeterminate, getTransitioningBarFillStyles());
+      addOnStyles = cx(indeterminate, transitioningFillStyles);
```

Comment:
nit: not sure of the performance implications on calling `cx` twice for the same styles. Might be more efficient to push to an array, and then spread that array into `cx` on L291